### PR TITLE
Implement pcountdown closebutton

### PR DESCRIPTION
**Pull request by dkent600**
## What was done

Implemented the pcountdown-closebutton.  This leverages pcountdown-circular to produce the control that will be present in popup notifications.

Demo here: https://prime-deals-dapp-git-pcountdown-closebutton-curvelabs.vercel.app/storybook/pCountdown-closebuttonDemo

countdown

## Testing

None

### CODE REVIEW

File: src/resources/elements/primeDesignSystem/pcountdown-closebutton/pcountdown-closebutton.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<template>
+  <div class="pcountdown-closebutton  ${ paused ? 'isPaused' : ''}">
```
Comment (IonelLupu) [inline src/resources/elements/primeDesignSystem/pcountdown-closebutton/pcountdown-closebutton.html:2]: what about using the <template> as the parent container element? This way we get rid of the `div` element

Comment (dkent600) [reply src/resources/elements/primeDesignSystem/pcountdown-closebutton/pcountdown-closebutton.html:2]: Done, note I also added a `closed` callback, cuz is more flexible and useful for the popup notification

**Review by IonelLupu — APPROVED** [review]

File: src/resources/elements/primeDesignSystem/pcountdown-closebutton/pcountdown-closebutton.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   */
+  @bindable.booleanAttr({ defaultBindingMode: bindingMode.twoWay}) running = true;
+
+  /**
+   * Is `cancelled` when the container manually stops the countdown, otherwise when
+   * the countdown hits 0.
+   * @param cancelled
+   */
+  countdownStopped(cancelled: boolean): void {
+    setTimeout(() => {
```
Comment (IonelLupu) [inline src/resources/elements/primeDesignSystem/pcountdown-closebutton/pcountdown-closebutton.ts:31]: why are these setTimeouts needed?

Comment (dkent600) [reply src/resources/elements/primeDesignSystem/pcountdown-closebutton/pcountdown-closebutton.ts:35]: I added comments in the code

**Pull request merged**